src/components/ScreenshotCard.js

```diff
@@ -4,7 +4,7 @@ import { Box, Image, Text } from 'grommet';
 const ScreenshotCard = ({ label, src }) => (
   <Box>
     <Image src={src} />
-    <Text alignSelf="center" color="altGrey" margin="medium">
+    <Text alignSelf="center" color="altGrey" margin={{ top: 'none' }}>
       {label}
     </Text>
   </Box>
```

Comment:
When the window is small I think the screenshots should be aligned in a column so they are easier to see